### PR TITLE
Fix description of default "verificationMethod"

### PR DESCRIPTION
**Pull request by peacekeeper**
Addresses https://github.com/w3c-ccg/vc-api/issues/67.

@clehner

### CODE REVIEW

File: components/VerifyOptions.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ components:
       properties:
         verificationMethod:
           type: string
-          description: The URI of the verificationMethod used for the proof. Default assertionMethod URI.
+          description: The URI of the verificationMethod used for the proof. If omitted a default verification method will be used.
```
Comment (msporny) [inline components/VerifyOptions.yml:22]: I'm wondering if we want to allow overriding the verificationMethod in the proof. It feels like a dangerous thing to do? I'm just wondering out loud right now... don't have a super strong feeling on this yet.

**Review by msporny — APPROVED** [review]
Note the concern w/ overriding verification method in VerifyOptions, otherwise, LGTM.

**Review by mkhraisha — APPROVED** [review]

File: components/IssueCredentialOptions.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ components:
           description: The type of the proof. Default is an appropriate proof type corresponding to the verification method.
         verificationMethod:
           type: string
-          description: The URI of the verificationMethod used for the proof. Default assertionMethod URI.
+          description: The URI of the verificationMethod used for the proof. If omitted a default verification method will be used.
```
Comment (TallTed) [inline components/IssueCredentialOptions.yml:25]:
```suggestion
          description: The URI of the verificationMethod used for the proof. If omitted, a default verification method will be used.
```

File: components/PresentCredentialOptions.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ components:
           description: The type of the proof. Default is an appropriate proof type corresponding to the verification method.
         verificationMethod:
           type: string
-          description: The URI of the verificationMethod used for the proof. Default assertionMethod URI.
+          description: The URI of the verificationMethod used for the proof. If omitted a default verification method will be used.
```
Comment (TallTed) [inline components/PresentCredentialOptions.yml:25]:
```suggestion
          description: The URI of the verificationMethod used for the proof. If omitted, a default verification method will be used.
```

File: components/VerifyOptions.yml (hunk 1 above)
Comment (TallTed) [inline components/VerifyOptions.yml:22]:
```suggestion
          description: The URI of the verificationMethod used for the proof. If omitted, a default verification method will be used.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by clehner — APPROVED** [review]

File: components/VerifyOptions.yml (hunk 1 above)
Comment (peacekeeper) [reply components/VerifyOptions.yml]: I agree this feels sensitive. We should merge this PR here and then create a new issue to discuss removing `verificationMethod` from `VerifyOptions`.

Comment (msporny) [reply components/VerifyOptions.yml]: I disagree that our default approach to identified security concerns is "merge it and discuss". People are implement to these APIs and we don't want to inject potential vulnerabilities into our specs.

**Review by msporny — CHANGES_REQUESTED** [review]
It's dangerous to override the `verificationMethod` in VerifyOptions. We should remove the possibility until it's clear that we have a use case for the feature.

**Comment by peacekeeper** [comment]
> I disagree that our default approach to identified security concerns is "merge it and discuss".

> It's dangerous to override the `verificationMethod` in VerifyOptions.

I agree, but what does it mean for this PR? This PR doesn't add or remove anything, it just fixes the description of what's already there.

Anyway, I just created this PR to remove it: https://github.com/w3c-ccg/vc-api/pull/253

**Review by msporny — APPROVED** [review]
Approving because PR #253 now exists.

**Pull request merged**